app/2.0/docs/devguide/dom-template.md

```diff
@@ -60,106 +60,184 @@ except for testing. For caveats about defining elements in the main document,
 see [main document definitions](registering-elements#main-document-definitions).
 {.alert .alert-info}
 
-### Specify a string template
+### Return an HTMLTemplateElement {#templateobject}
 
-As an alternative to specifying the element's template in markup, you can specify a string template
-by creating a static `template` getter that returns a string.
+As an alternative to specifying the element's template in markup, you can create a static
+`template` getter that returns an HTMLTemplateElement.
 
 This getter is called _once_, when the first instance of the element is upgraded.
 
 ```js
 class MyElement extends Polymer.Element {
 
   static get template() {
-    return `<style>:host { color: blue; }</style>
+    return Polymer.html`<style>:host { color: blue; }</style>
        <h2>String template</h2>
-       <div>I've got a string template!</div>`
+       <div>I've got a string template!</div>`;
   }
 }
 customElements.define('my-element', MyElement);
 ```
 
-When using a string template, the element doesn't need to provide an `is` getter (however, the tag
-name still needs to  be  passed as the first argument to `customElements.define`).
+Alternatively, declare a variable called `html` to hold the `Polymer.html` object, and
```

Comment:
I feel like this should be listed goal-first. It's a lot of text to read through to figure out that you'd want to do this to enable syntax highlighting.